Jenkinsfile

```diff
@@ -223,6 +223,31 @@ def job = {
         }
     }
 
+    if (!config.isPrJob && config.release) {
+        stage('Publish Maven Artifacts') {
+            writeFile file: settingsFile, text: settings
+            dir('ksql-db') {
+                withCredentials([usernamePassword(credentialsId: 'Jenkins Nexus Account', passwordVariable: 'NEXUS_PASSWORD', usernameVariable: 'NEXUS_USERNAME')]) {
+                    withDockerServer([uri: dockerHost()]) {
```

Comment:
is the dockerserver necessary?

Reply:
pretty sure we need that since the build is running in a docker container.